targets/TARGET_ARM_SSG/TARGET_MUSCA_A1/PeripheralNames.h

```diff
@@ -0,0 +1,46 @@
+/* mbed Microcontroller Library
+ * Copyright (c) 2006-2017 ARM Limited
```

Comment:
can you fix the year (2018) in all new files. Worth also adding SPDX identifier: `SPDX-License-Identifier: Apache-2.0`

Reply:
Fix commit has been submitted.